examples/host_commands.h

```diff
@@ -48,6 +49,68 @@ struct hoth_response_flash_spi_info {
   uint8_t sr1, sr2;
 } __hoth_align1;
 
+/* Returns the architecturally defined application firmware versions.
+ * The request payload is zero bytes, and the response payload is a struct
+ * get_version_host_cmd_response.  variant_ is always 0x1337; all other fields
+ * are populated from the RW firmware image header.
+ */
+#define EC_PRV_CMD_HAVEN_GET_APP_VERSION 0x0002
+
+#define EC_CMD_BOARD_SPECIFIC_BASE 0x3E00
+#define EC_CMD_BOARD_SPECIFIC_LAST 0x3FFF
+
+// NOTE: All further commands in this file are offset by
+// EC_CMD_BOARD_SPECIFIC_BASE.
+
+/**
+ * The identifier for the SecurityV2 host command. All SecurityV2 commands are
+ * handled by a single host command handler. Specific SecurityV2 commands are
+ * identified by the "major_command" and "minor_command" fields of a command's
+ * request header.
+ */
+#define EC_PRV_CMD_HOTH_SECURITY_V2 0x0003
```

Comment:
nit: HOTH_PRV_CMD_HOTH rather than EC_PRV_CMD_HOTH, to match other constants throughout this file